.travis.yml

```diff
@@ -100,9 +109,15 @@ before_script:
   #- sudo chmod +s $(which gdb)
 
 script:
-  - echo 'variables_order=EGPCS' >> "$(phpenv root)/versions/$(phpenv version-name)/etc/php.ini"
+  - echo -e 'variables_order=EGPCS\nenable_dl=true' >> "$(phpenv root)/versions/$(phpenv version-name)/etc/php.ini"
```

Comment:
Could you please create an ini file like:
```ini
variables_order=EGPCS
```
and then add this file via:
```yaml
- phpenv config-add unit-tests/ci/zephir.ini
```

This approach will allow us to add more settings in future to this file without using echo. Also as you can see the **`enable_dl`** is deprecated feature and will be removed in the future: http://php.net/manual/en/info.configuration.php Why do we need this?

Reply:
The current (unpatched) approach already uses a separate ini file (though admittedly I commented that part out, above) and a change to the core ini to make things function as needed for these tests. This approach is very inflexible and non-portable.

What I mean, here, is that the current approach requires changes to the system-wide configuration of PHP itself, something which isn't always desirable, or in some cases even possible. This prevents running tests locally, meaning Travis becomes the only testing platform available, and every change will likely result in several test runs to resolve any problems. Alternately, users who contribute to Zephir dev will need to have the `test` extension loaded across their entire system, at least for the duration of tests, and are responsible for remembering to add and/or remove this entry every time manually.

The primary purpose of this PR is to remove any need to modify the ini files at all to successfully run tests. One option is to load the extension directly via the CLI, using `php -d 'extension=test' ./unit-test/phpunit ...`, but this approach actually leads to memory leaks that cause Valgrind to fail the tests. So I've opted for the alternative where the test bootstraps load the extension themselves, assuming it isn't loaded already, which works perfectly in every case, and allows tests to be run locally as well. (The PHPUnit docs say that it supports the `-d` option to its CLI command, but I wasn't able to get that working in any way, so here we are.)